From 1e280e1b30aa5ddafbc9967e31a64f771fcaa080 Mon Sep 17 00:00:00 2001
From: Artur Opalinski <ArturX.Opalinski@intel.com>
Date: Wed, 17 Jun 2020 12:01:20 +0200
Subject: [PATCH 7/7] Enable flags to emulate PCH-uart in PCH-SKL as legacy
 device 16550A      CONFIG_SERIAL_8250_DW=y      CONFIG_MFD_INTEL_LPSS_ACPI=y 
     CONFIG_MFD_INTEL_LPSS_PCI=y

---
 .config | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/.config b/.config
index 7a731d985816..6f5932510096 100644
--- a/.config
+++ b/.config
@@ -2301,7 +2301,7 @@ CONFIG_SERIAL_8250_MANY_PORTS=y
 CONFIG_SERIAL_8250_SHARE_IRQ=y
 CONFIG_SERIAL_8250_DETECT_IRQ=y
 CONFIG_SERIAL_8250_RSA=y
-# CONFIG_SERIAL_8250_DW is not set
+CONFIG_SERIAL_8250_DW=y
 # CONFIG_SERIAL_8250_RT288X is not set
 CONFIG_SERIAL_8250_LPSS=y
 CONFIG_SERIAL_8250_MID=y
@@ -2756,8 +2756,8 @@ CONFIG_BCMA_POSSIBLE=y
 # CONFIG_MFD_INTEL_QUARK_I2C_GPIO is not set
 # CONFIG_LPC_ICH is not set
 # CONFIG_LPC_SCH is not set
-# CONFIG_MFD_INTEL_LPSS_ACPI is not set
-# CONFIG_MFD_INTEL_LPSS_PCI is not set
+CONFIG_MFD_INTEL_LPSS_ACPI=y
+CONFIG_MFD_INTEL_LPSS_PCI=y
 # CONFIG_MFD_JANZ_CMODIO is not set
 # CONFIG_MFD_KEMPLD is not set
 # CONFIG_MFD_88PM800 is not set
-- 
2.17.1

